pubspec.yaml

```diff
@@ -9,6 +9,7 @@ environment:
 
 dependencies:
   source_span: ^1.8.0
+  path: ^1.8.0
```

Comment:
This should be under dev_dependencies since it's only used in tests, correct?

Reply:
Yes, fixed!